open-company/open-company-web
# Fix go to digest button

From the card https://trello.com/c/f6R86N9a

Abstract: https://share.goabstract.com/9c535b8d-2121-4c3a-bb6f-168feb91d355

To test:
- visit our home or /slack or /about when you are logged in
- do you see the new Go to digest button with the new icon? Good
- the design look good? Good
- merge
